timesketch/frontend-ng/src/components/LeftPanel/SigmaRules.vue

```diff
@@ -76,11 +101,8 @@ export default {
     },
   },
   created() {
-    ApiClient.getSigmaList()
-      .then((response) => {
-        this.sigmaRules = response.data.objects
-      })
-      .catch((e) => {})
+    console.log('SigmaRules created')
```

Comment:
Can probably remove this debug statement?

Reply:
sure sure - done